### PR TITLE
skipDevice flow on buy

### PR DESCRIPTION
**Pull request by machard**
https://ledgerhq.atlassian.net/browse/LL-3704

### CODE REVIEW

**Comment by juan-cortes** [comment]
Would be great to have some screenshots of this if you don't mind @machard

**Comment by machard** [comment]
yep,
Capture d’écran 2020-12-22 à 10 36 24
Capture d’écran 2020-12-22 à 10 35 43

**Review by juan-cortes — APPROVED** [review]
lgtm but check the wording with product just to be sure

File: src/locales/en/common.json [hunk 1]
```diff
@@ -3081,6 +3081,12 @@
         "title": "No {{currency}} account",
         "description": "Please add an account before buying {{currency}}.",
         "CTAButton": "Add account"
+      },
+      "skipDeviceVerification": {
+        "confirm": "Confirm",
+        "cancel": "Cancel",
+        "address": "Address for {{currency}} account",
+        "warning": "You address was not confirmed on your Ledger Device. Please verify it for security"
```
[the remark this answers is not in the document]
Comment (dasilvarosa) [reply src/locales/en/common.json:3089]: Correct. “Your” please

**Comment by gre** [comment]
conflicts. feel free to ask @LFBarreto  (dark mode related i think)

**Review by gre — APPROVED** [review]

**Review by nabil-brn — CHANGES_REQUESTED** [review]
It crashes at boot on iOS and Android.

---
5afd3afa5 (HEAD -> pr/machard/1517, machard/LL-3704) Merge branch ‘develop’ into LL-3704

**Review by nabil-brn — APPROVED** [review]
✓ QA-ed

---
bfda22a71 (HEAD -> LL-3704, machard/LL-3704) fix colors

**Review by gre — APPROVED** [review]

**Pull request merged**